src/solvers/smt2_incremental/convert_expr_to_smt.cpp

```diff
@@ -325,8 +325,12 @@ convert_expr_to_smt(const binary_relation_exprt &binary_relation)
 
 static smt_termt convert_expr_to_smt(const plus_exprt &plus)
 {
-  UNIMPLEMENTED_FEATURE(
-    "Generation of SMT formula for plus expression: " + plus.pretty());
+  for(const auto &op : plus.operands())
```

Comment:
⛏️ `op` should be `operand` given the coding standards guideline to "avoid abbreviations". `op` is a particularly ambiguous abbreviation given that you need to read the context to know if it is an "operator" / "operation" or "operand".

Reply:
Looks good to me.